README.md

```diff
@@ -79,3 +79,5 @@ Synapse developers (assuming a Unix-like shell):
     ```shell
     git push origin tag v$version
     ```
+
+ 7. [Create a *GitHub release*](https://github.com/matrix-org/synapse-bind-sydent/releases/new), based on the tag you just pushed. 
```

Comment:
Don't we need step 8 also?

> If applicable: Create a source distribution and upload it to PyPI:

Or do we not upload this anywhere?

Reply:
We currently don't, and don't plan to really support this module for public consumption (we should add a warning to this effect on the README as well.)

Those that currently use this module are happy to pull from GitHub, and publishing to PyPI at the moment will probably only be a signal that this is supported.